### PR TITLE
work on the main services section in both HTML and CSS

### PR DESCRIPTION
**Pull request by omaralrabi**
rel #2
I edited both files. Was able to create a main services section that consists of 3 main sections:

1. Buy.
2. Sell.
3. Rent.
I used the assigned team colors, added a hover, making it interactive, and all of that was inspired by a real estate website.

Capture

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,52 +1,156 @@
...
+  border: solid thick rgba(255, 255, 255, 0.548) 1px;
+  border-radius: 6px;
+  box-shadow: rgb(209 209 213) 0px 5px;
+  text-align: center;
+}
+.Buy_Button{
+  padding: 10px;
+  font-family: 'Montserrat', sans-serif;
+}
+.Buy_Button:hover{
```
Comment (AbdallahGot) [inline style.css:117]: add this code( cursor: pointer; ) here 
to change the cursor to a pointer when the user puts the mouse cursor on it.

File: style.css [hunk 2]
```diff
@@ -1,52 +1,156 @@
...
+  padding: 10px;
+  color: white;
+  background-color: #0C4499;
+  }
+.Sell_Button{
+  padding: 10px;
+  font-family: 'Montserrat', sans-serif;
+}
+.Sell_Button:hover{
```
Comment (AbdallahGot) [inline style.css:126]: add this code( cursor: pointer; ) here 
to change the cursor to a pointer when the user puts the mouse cursor on it.

File: style.css [hunk 3]
```diff
@@ -1,52 +1,156 @@
...
+  padding: 10px;
+  color: white;
+  background-color: #0C4499;
+}
 
+.Rent_Button{
+  padding: 10px;
+  font-family: 'Montserrat', sans-serif;
+}
+.Rent_Button:hover{
```
Comment (AbdallahGot) [inline style.css:136]: add this code( cursor: pointer; ) here 
to change the cursor to a pointer when the user puts the mouse cursor on it.

**Review by AbdallahGot — COMMENTED** [review]
good job omer, but I want you to look at this image I think the three blocks aren't at the same line.
try to adjust it.

**Review by AbdallahGot — APPROVED** [review]

**Review by sajaabusafia — CHANGES_REQUESTED** [review]
I suggest adding this, so the buttons look better
`.services input   {
      margin: 30px;
      background-color: white;
      width: 150px;
      border: 1px solid #0C4499;
      border-radius: 20px;
  }`

File: style.css [hunk 4]
```diff
@@ -1,52 +1,156 @@
-*{
-    margin:0;
-    padding:0;
-    box-sizing:border-box;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
```
Comment (sara219) [inline style.css:5]: duplicate code!
we already have the reset code in the `main`

https://github.com/GSG-FC02/Real-Estate-Website/blob/3ca32faf44e45e31b8a7f179bb2c49a6cbb04009/style.css#L1-L5

you just need to add the font-family to it

File: style.css [hunk 5]
```diff
@@ -1,52 +1,156 @@
...
+  background-color: white;
+  padding: 70px 50px;
+  border: solid thick rgba(255, 255, 255, 0.548) 1px;
+  border-radius: 6px;
+  box-shadow: rgb(209 209 213) 0px 5px;
+  text-align: center;
+}
+.Buy_Button{
+  padding: 10px;
+  font-family: 'Montserrat', sans-serif;
```
Comment (sara219) [inline style.css:115]: You repeat the same line for the 3-buttons  (rent, buy and sell)

https://github.com/GSG-FC02/Real-Estate-Website/blob/1c0a5a1e2031018d55da294c092c089ed8d3068f/style.css#L132-L135

https://github.com/GSG-FC02/Real-Estate-Website/blob/1c0a5a1e2031018d55da294c092c089ed8d3068f/style.css#L122-L125

while you can merge it in one rule-set 
give the 3-box same class name and give it a rule set at once

File: style.css [hunk 6]
```diff
@@ -1,52 +1,156 @@
...
+  }
+.Sell_Button{
+  padding: 10px;
+  font-family: 'Montserrat', sans-serif;
+}
+.Sell_Button:hover{
+  padding: 10px;
+  color: white;
+  background-color: #0C4499;
+}
```
Comment (sara219) [inline style.css:130]: The same goes there, you repeat the hover thing for 3 time

https://github.com/GSG-FC02/Real-Estate-Website/blob/1c0a5a1e2031018d55da294c092c089ed8d3068f/style.css#L117-L120

https://github.com/GSG-FC02/Real-Estate-Website/blob/1c0a5a1e2031018d55da294c092c089ed8d3068f/style.css#L126-L130

https://github.com/GSG-FC02/Real-Estate-Website/blob/1c0a5a1e2031018d55da294c092c089ed8d3068f/style.css#L136-L140

File: style.css [hunk 7]
```diff
@@ -1,52 +1,156 @@
...
+  flex-direction: column;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: stretch;
+  -ms-flex-align: stretch;
+  align-items: stretch;
+  text-align: center;
+  padding: 35px;
+}
```
Comment (sara219) [inline style.css:89]: what is the purpose of that code! 

it's useless, you don't have a class called `elementor-flip-box__layer__overlay`
and I don't know why you write it!

File: style.css [hunk 8]
```diff
@@ -1,52 +1,156 @@
...
 
-.header_nav_list a:hover{
-    color: rgb(241, 245, 8);
-}
\ No newline at end of file
+.services:hover {
+  content: '';
+  box-shadow: rgb(209 209 213) 0px 5px 15px;
+  opacity: 0;
+  opacity: 1;
```
Comment (sara219) [inline style.css:150]: 0 or 1 🙃
delete the first one, it's an unnecessary line,

File: style.css [hunk 9]
```diff
@@ -1,52 +1,156 @@
...
+/* Services Section Styles */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+.buy-image {
+  margin: auto;
+  width: 50px;
+}
```
Comment (sara219) [inline style.css:62]: This code does not work

File: style.css [hunk 10]
```diff
@@ -1,52 +1,156 @@
...
+  text-align: center;
+  padding: 35px;
+}
+
+.services_section {
+  background-color: rgb(202 204 206 / 16%);
+  width: 100%;
+  height: 630px;
+}
+.services_section_container {
```
Comment (sara219) [inline style.css:96]: These boxes do not have the same height

Screenshot from 2021-03-04 20-38-08

**Review by sara219 — CHANGES_REQUESTED** [review]
👇

**Comment by omaralrabi** [comment]
I took all the feedback into careful consideration, edited the files on my branch, and will push them into the main branch once again and request your review.

**Pull request closed without merging**